docs/tutorials/python.md

```diff
@@ -0,0 +1,294 @@
+# Tutorial: Doing python development with pixi
+
+In this tutorial we will show you how to create a simple python project with pixi. We will show some of the features that pixi provides, that are currently
+not a part of `pdm`, `poetry` etc.
+
+## Why is this useful?
+Pixi builts upon on the conda ecosystem, which allows you to create a python environment with all the dependencies you need. This is especially useful when you are working with multiple python interpreters and bindings to C and C++ libraries. Let's give it a go! E.g GDAL from PyPI does not have the binary C dependencies, but the conda package does.
+
+## pixi.toml and pyproject.toml
+We support two manifests formats, `pyproject.toml` and `pixi.toml`. In this tutorial we will use the `pyproject.toml` format. Because it is the most common format for python projects.
+
+## Let's get started
+
+Lets start out by making a directory and creating a new `pyproject.toml` file.
+
+```shell
+pixi init pixi_py --pyproject
+```
+
+This gives you the following pyproject.toml:
+
+```toml
+[project]
+name = "pixi_py"
+version = "0.1.0"
+description = "Add a short description here"
+authors = [{name = "Tim de Jager", email = "tim@prefix.dev"}]
+requires-python = ">= 3.11"
+dependencies = []
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.pixi.project]
+channels = ["conda-forge"]
+platforms = ["osx-arm64"]
+
+[tool.pixi.pypi-dependencies]
+mypy = { path = ".", editable = true }
+
+[tool.pixi.tasks]
+```
+
+Let's add the python project to the tree:
+
+```shell
+mkdir pixi_py
+touch pixi_py/__init__.py
+```
+
+We now have the following directory structure:
+
+```shell
+.
+├── pixi_py
+│   ├── __init__.py
+└── pyproject.toml
+```
+
+We've uses a flat-layout here but we support both.
+
+### What's in the pyproject?
+
+Okay, so let's have a look at what's sections have been added.
+
+These things were added to the pyproject.toml file:
+
+```toml
+# 1. Main pixi entry
+[tool.pixi.project]
+channels = ["conda-forge"]
+# This is your machine platform by default
+platforms = ["osx-arm64"]
+
+# 2. Editable installs
+[tool.pixi.pypi-dependencies]
+pixi_py = { path = ".", editable = true }
+
+# 3. Environments
+[tool.pixi.environments]
+default = { solve-group = "default" }
+test = { features = ["test"], solve-group = "default" }
```

Comment:
```suggestion
```
I would skip them here as you only explain the first 2 here.

Reply:
Yeah I moved it that was a mistake :)